python/setup.py

```diff
@@ -131,6 +123,12 @@ def run(self):
     # Generate necessary .proto file if it doesn't exist.
     generate_proto("../src/google/protobuf/descriptor.proto")
     generate_proto("../src/google/protobuf/compiler/plugin.proto")
+    generate_proto("../src/google/protobuf/timestamp.proto")
```

Comment:
This list is not complete. See the complete list here:
https://github.com/google/protobuf/blob/master/src/Makefile.am#L35